openmoc/krylov.py

```diff
@@ -8,18 +8,17 @@
 # @date June 30, 2015
 
 import sys
-import copy
+
 import numpy as np
-import scipy.sparse.linalg as linalg
```

Comment:
Is there any reason this was moved into the functions and imported in two places? It seems more appropriate to keep the import here.